packages/teams-js/test/public/media.spec.ts

```diff
@@ -46,20 +47,30 @@ describe('media', () => {
     jest.clearAllMocks();
   });
 
+  describe('isSupported', () => {
+    it('returns true if media is supported', () => {
+      Object.defineProperty(runtime.supports, 'media', { value: true });
```

Comment:
I think this should be setting the value to '{}', e.g.,:
`Object.defineProperty(runtime.supports, 'media', { value: {} });`
because that is closer to how the actual code sets the value when a capability is supported (e.g., it just checks that it's not undefined).

Reply:
I'm resolving this since we aren't using defineProperty anymore

Reply:
I checked and it looks like these changes have now been made